azurerm/internal/services/postgres/tests/resource_arm_postgresql_server_test.go

```diff
@@ -20,15 +21,37 @@ func TestAccAzureRMPostgreSQLServer_basicNinePointFive(t *testing.T) {
 		CheckDestroy: testCheckAzureRMPostgreSQLServerDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccAzureRMPostgreSQLServer_basicNinePointFive(data),
+				Config: testAccAzureRMPostgreSQLServer_basic(data, "9.5"),
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMPostgreSQLServerExists(data.ResourceName),
-					resource.TestCheckResourceAttr(data.ResourceName, "administrator_login", "acctestun"),
-					resource.TestCheckResourceAttr(data.ResourceName, "version", "9.5"),
-					resource.TestCheckResourceAttr(data.ResourceName, "ssl_enforcement", "Enabled"),
 				),
 			},
-			data.ImportStep("administrator_login_password"),
+			data.ImportStep("administrator_login_password", "create_mode"),
```

Comment:
since this is ForceNew and defaults - we'll probably want to set this during the import func?